requirements.txt

```diff
@@ -1,4 +1,4 @@
-cryptography==1.8.2
+cryptography>=1.8.1,<1.10
```

Comment:
Well, the prior version of http-ece is limited to 1.8.1. vapid doesn't use http-ece, so we're at liberty over there. 

martin landed the patch to move to 1.9, but hasn't promoted the package on pypi yet.